Workbooks/SqlDatabase/SQL Database Resource Limits/SQL Database Metrics.workbook

```diff
@@ -0,0 +1,398 @@
+        "timeContextFromParameter": "TimeRange",
+        "resourceType": "microsoft.sql/servers/databases",
+        "resourceParameter": "Database",
+        "metrics": [
+          {
+            "namespace": "microsoft.sql/servers/databases",
+            "metric": "microsoft.sql/servers/databases--sessions_percent",
+            "aggregation": 4,
+            "chartType": 2,
+            "color": "blue"
+          }
+        ]
+      },
+      "conditionalVisibility": null,
+      "halfWidth": true
+    },
+    {
+      "type": 10,
+      "content": {
+        "chartId": "workbook459ff374-8839-49fb-9d33-faa8570ac2f3",
+        "version": "MetricsItem/1.0",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 0,
+          "endTime": null,
+          "createdTime": "2018-07-17T21:26:20.076Z",
+          "isInitialTime": false,
+          "grain": 1,
+          "useDashboardTimeRange": false
+        },
+        "timeContextFromParameter": "TimeRange",
+        "resourceType": "microsoft.sql/servers/databases",
+        "resourceParameter": "Database",
+        "metrics": [
+          {
+            "namespace": "microsoft.sql/servers/databases",
+            "metric": "microsoft.sql/servers/databases--workers_percent",
+            "aggregation": 4,
+            "chartType": 2,
+            "color": "blue"
+          }
+        ]
+      },
+      "conditionalVisibility": null,
+      "halfWidth": true
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "When encountering high session or worker utilization, mitigation options include:\n* Increasing the service tier or performance level of the database or elastic pool. See [Scale single database resources](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-single-database-scale) and [Scale elastic pool resources](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool-scale).\n* Optimizing queries to reduce the resource utilization of each query if the cause of increased worker utilization is due to contention for compute resources. For more information, see [Query Tuning/Hinting](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-performance-guidance#query-tuning-and-hinting).\n\n\n When encountering high session or worker utilization, mitigation options include:\n* Increasing the service tier or performance level of the database. See [Scale single database resources](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-single-database-scale) and [Scale elastic pool resources](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool-scale).\n* Optimizing queries to reduce the resource utilization of each query if the cause of increased worker utilization is due to contention for compute resources. For more information, see [Query Tuning/Hinting](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-performance-guidance#query-tuning-and-hinting)."
+      },
```

Comment:
this section here "when encountering high session or worker utilization..." is duplicated?